src/scss/init/unity-layout/unity-layout.scss

```diff
@@ -0,0 +1,69 @@
+@import "../../init";
+
+
+@mixin unity-layout($values) {
+  @if type-of($values) == "list" {
+    @if (length($values) == 2) {
+      $value-1: nth($values, 1);
+      $value-2: nth($values, 2);
+
+      @if ($value-1 > $value-2) {
+        .content-sidebar-wrap {
+          width: 100%;
+        }
+
+        #content {
+          width: $value-1;
+        }
+
+        #sidebar-first {
+          width: $value-2;
+        }
+      }
+      @else {
+        .content-sidebar-wrap {
+          width: $value-2;
+        }
+
+        #content {
+          width: 100%;
+        }
+
+        #sidebar-second {
+          width: $value-1;
+        }
+      }
+    }
+    @else if (length($values) == 3) {
+      $value-1: nth($values, 1);
```

Comment:
need to be careful with these values as they should be set as percentage values for each column, however to get them to compile correctly we might need raw numbers, might need to do some logic around this to test what the value is, so if for example the developer forgets the % is there enough in the code to return a legitimate css rule.